inscriptions/ISic020087.xml

```diff
@@ -92,8 +92,11 @@
                             <handNote>
                                 <!--  Glyph chart: fragm.a) epsilon19(?); lambda08 or upsilon03 or Elymian symbol (id: num03; see nos.83-84 Agostiniani 1977); pi11 (?unclear); fragm.b) alpha09; delta05.-->
                                 <locus from="line1" to="line1">Line 1</locus>
-                                <dimensions type="letterHeight">
-                                    <height unit="mm"/>
+                                <dimensions xml:id="faceA" type="letterHeight">
```

Comment:
I'm not sure about this use of xml:id here, rather than locus? leave it for now, but it seems to be divergent from our practice elsewhere?